py/qqa/run.py

```diff
@@ -285,6 +285,14 @@ def make_plots(infile, outdir):
     htmlfile = '{}/qa-summary-{:08d}.html'.format(outdir, expid)
     webpages.summary.write_summary_html(htmlfile, plot_components)
     print('Wrote {}'.format(htmlfile))
+    from qqa.webpages import plotimage
+    if (preprocdir is not None):
+        if cameras is None:
+            cameras = which_cameras(rawfile)
+        for camera in cameras:
+            input = os.path.join(preprocdir, "preproc-{}-{:08d}.fits".format(camera, expid))
+            output = os.path.join(outdir, "preproc-{}-{:08d}-4x.html".format(camera, expid))
+            plotimage.write_image_html(input, output, 4)
```

Comment:
Nice to include the option of limiting which cameras to use, but rather than passing in the rawfile, let's have it default to processing every preproc it can find.  e.g. something like
```python
import glob
for preprocfile in glob.glob(os.path.join(preprocdir, 'preproc-*-*.fits')):
    if cameras is not None:
        # ... derive camera name by parsing the preproc file name ...
        if camera not in cameras:
            continue

    # proceed with plotimage
```

Otherwise this PR looks good.  Thanks.

Reply:
Alright, I think I fixed this in the latest commit (3b3c725).